src/components/SpaceItem.vue

```diff
@@ -0,0 +1,33 @@
+<script lang="ts" setup>
+import { Space } from '@/types';
+import { useSpacesStore } from '@/stores/spaces';
```

Comment:
```suggestion
import { useSpacesStore } from '@/stores/spaces';
import { Space } from '@/types';
```

nit. we tend to put `@/types` imports at the bottom (here and in other files in this PR).